src/contracts.rs

```diff
@@ -128,6 +140,18 @@ pub fn read_pubkeys_from_multisig_redeemscript(
     Ok((pubkey1.unwrap(), pubkey2.unwrap()))
 }
 
+// Added as a helper, maybe useful later.
+pub fn read_pubkeys_from_contract_reedimscript(
+    contract_script: &Script,
+) -> Result<(PublicKey, PublicKey), &'static str> {
+    let script_bytes = contract_script.to_bytes();
+
+    let hashpub = PublicKey::from_slice(&script_bytes[27..60]).map_err(|_| "Bad pubkey data")?;
+    let timepub = PublicKey::from_slice(&script_bytes[65..98]).map_err(|_| "Bad pubkey data")?;
+
+    Ok((hashpub, timepub))
+}
+
```

Comment:
I'd say leave this out. Right now as its unused it just clutters up the codebase. If it turns out to be needed later then it can be added then.

Reply:
Moving this to test module.